docs/BR.md

```diff
@@ -1633,66 +1638,61 @@ By issuing the Certificate, the CA represents that it followed the procedure set
 Subject attributes MUST NOT contain only metadata such as '.', '-', and ' ' (i.e. space) characters, and/or any other indication that the value is absent, incomplete, or not applicable.
 
 #### 7.1.4.2.1 Subject Alternative Name Extension
-Certificate Field: extensions:subjectAltName
-Required/Optional: Required
-Contents: This extension MUST contain at least one entry. Each entry MUST be either a dNSName containing the Fully-Qualified Domain Name or an iPAddress containing the IP address of a server. The CA MUST confirm that the Applicant controls the Fully-Qualified Domain Name or IP address or has been granted the right to use it by the Domain Name Registrant or IP address assignee, as appropriate.
+
+__Certificate Field:__ `extensions:subjectAltName`  
```

Comment:
Our other sections-of-emphasis use `**` rather than `__` - I'm curious if that was similarly intentional here?

Reply:
Hmmmm. Not intentional, no—not sure what my brain was doing at the time. Pandoc doesn't care, but it's a good idea to be consistent. For visibility with all the asterisks in the document, I might recommend converting everything to double-underscore, but I'm flexible on this one. Thoughts?